packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -25,6 +25,27 @@ export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartSt
     },
     legendContainer: {
       paddingTop: '4px'
+    },
+    hoverCardTextStyles: {
+      fontFamily: 'Segoe UI',
+      fontSize: '12px',
+      lineHeight: '14px'
+    },
+    hoverCardDataStyles: {
+      color: legendColor === '' ? '#000000' : legendColor,
```

Comment:
Use `theme.palette.black`. Also, this can be simplified to `legendColor || theme.palette.black`.